### DISCUSSION TITLE
How to uninstall php from macOS 10.15 and after that install latest  version of php 8.2

### DISCUSSION THREAD

**Discussion by pawanweber** [community/community#123487] [Programming Help]
### Body

In my mac php already installed. when i checking php version help of terminal by command '**php -v**' it showing **php@7.3** but at '**http://localhost/dashboard/phpinfo.php**' php version showing **php@8.2.4**

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by liker912** [comment]
Hi! Looks like you have multiple PHP instalations on your mac. I recommend to does not remove the php@7.3.

Try to set php 8.2 as a default version 

```
# unlink the old
brew unlink php@7.3

# set the new
brew link --overwrite --force php@8.2
```

If in some cases you will need the 7.3 php version you can add an alias.
Open your shell configuration file (~/.bash_profile, ~/.bashrc, or ~/.zshrc)

add or change alias: 
```
alias php73='/usr/local/opt/php@7.3/bin/php'
```

and use it by php73 script.php

**Reply by ebndev** [reply]
Hey @pawanweber - did you get a chance to try the advice above / find a solution?
